### PR TITLE
added c-sharp heading and two of its courses in free-courses-hi.md

### PR DESCRIPTION
**Pull request by swastikgithub09**
## What does this PR do?
Add resource(s) 

### Description
added c-sharp(#) in index and two courses under it
### Why is this valuable (or not)?
previously, no c# tutorial was there 
### How do we know it's really free?
free on youtube
### For book lists, is it a book? For course lists, is it a course? etc.
courses
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -69,6 +70,13 @@
 * [C Programming Tutorials](https://www.youtube.com/playlist?list=PLiOa6ike4WAHH3HyPUu6pUG-r0LApvW-l) - Vikas Pandey, easytuts4you
 
 
+### <a id="csharp"></a>C\#
+
+* [C# PROGRAMMING LANGUAGE (CONSOLE APPLICATIONS)](https://www.youtube.com/playlist?list=PLX07l0qxoHFLZftsVKyj3k9kfMca2uaPR) -
```
Comment (eshellman) [inline courses/free-courses-hi.md:75]: no all caps titles, even if it's from the resource

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -69,6 +70,13 @@
 * [C Programming Tutorials](https://www.youtube.com/playlist?list=PLiOa6ike4WAHH3HyPUu6pUG-r0LApvW-l) - Vikas Pandey, easytuts4you
 
 
+### <a id="csharp"></a>C\#
+
+* [C# Programming language (Console Applications)](https://www.youtube.com/playlist?list=PLX07l0qxoHFLZftsVKyj3k9kfMca2uaPR) -
```
Comment (davorpa) [inline courses/free-courses-hi.md:75]: - Don't leave the author chunk empty

```suggestion
* [C# Programming language (Console Applications)](https://www.youtube.com/playlist?list=PLX07l0qxoHFLZftsVKyj3k9kfMca2uaPR) - Learning Never Ends
```

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -69,6 +70,13 @@
 * [C Programming Tutorials](https://www.youtube.com/playlist?list=PLiOa6ike4WAHH3HyPUu6pUG-r0LApvW-l) - Vikas Pandey, easytuts4you
 
 
+### <a id="csharp"></a>C\#
+
+* [C# Programming language (Console Applications)](https://www.youtube.com/playlist?list=PLX07l0qxoHFLZftsVKyj3k9kfMca2uaPR) - 
+Learning Never Ends (YouTube)
```
Comment (davorpa) [inline courses/free-courses-hi.md:76]: Don't leave the author chunk empty

```suggestion
* [C# Programming language (Console Applications)](https://www.youtube.com/playlist?list=PLX07l0qxoHFLZftsVKyj3k9kfMca2uaPR) - Learning Never Ends (YouTube)
```

File: courses/free-courses-hi.md [hunk 4]
```diff
@@ -69,6 +70,13 @@
...
+* [C# Tutorial In Hindi](https://www.youtube.com/watch?v=SuLiu5AK9Ps) - CodeWithHarry (YouTube)
```
Comment (davorpa) [inline courses/free-courses-hi.md:77]: No YT channel anotations

```suggestion
* [C# Tutorial In Hindi](https://www.youtube.com/watch?v=SuLiu5AK9Ps) - CodeWithHarry
```

Comment (davorpa) [reply courses/free-courses-hi.md]: Solved at 9c7f567e9fd09c7c7f34926d44c97dc267d6e94b

File: courses/free-courses-hi.md (hunk 3 above)
Comment (davorpa) [reply courses/free-courses-hi.md]: Solved at 9c7f567e9fd09c7c7f34926d44c97dc267d6e94b

**Review by davorpa — APPROVED** [review]

**Pull request merged**